pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -530,6 +558,17 @@ func (r *reconciler) reconcileSecrets(ctx context.Context, data reconcileData) e
 		}
 	}
 
+	if r.userClusterMLA.Logging {
+		creators = []reconciling.NamedSecretCreatorGetter{
+			promtail.SecretCreator(promtail.Config{
+				MLAGatewayURL: fmt.Sprintf("http://%s:%d", r.clusterURL.Hostname(), r.userClusterMLA.MLAGatewayPort),
```

Comment:
This URL is used for the Loki Promtail in User cluster to push logs to the Loki in the Seed. Since we haven't deployed and enabled https for the Loki in Seed, so for now, we just put `http` here. In the future, we will use `https`.